packages/puppeteer-core/src/injected/util.ts

```diff
@@ -64,5 +64,5 @@ export const checkVisibility = (
 
 function isBoundingBoxVisible(element: Element): boolean {
```

Comment:
nit: rename isBoundingBoxVisible to something that does not have word Visible in it?